extensions/http-provisioner/src/main/java/org/eclipse/dataspaceconnector/transfer/provision/http/webhook/HttpProvisionerWebhookApiController.java

```diff
@@ -34,6 +35,7 @@
 @Consumes({ MediaType.APPLICATION_JSON })
 @Produces({ MediaType.APPLICATION_JSON })
 @Path("/callback")
+@Tag(name = "HTTP Provisioner Webhook")
```

Comment:
We agreed on putting the swagger annotations on the `*Api` interface, not on the `*ApiController` implementation.